<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AndroidDeployStepWidget</class>
 <widget class="QWidget" name="AndroidDeployStepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>145</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="devicesQtLibs">
     <property name="toolTip">
      <string>Use Qt libraries from device</string>
     </property>
     <property name="text">
      <string>Use Qt libraries from device</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="deployQtLibs">
       <property name="toolTip">
        <string>Push local Qt libraries to device.
You must have Qt libraries compiled for that platform</string>
       </property>
       <property name="text">
        <string>Deploy local Qt libraries</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="useLocalQtLibs">
       <property name="toolTip">
        <string>Check this option to force the application to use local Qt libraries instead of system libraries.</string>
       </property>
       <property name="text">
        <string>Use local Qt libraries</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cleanLibsPushButton">
       <property name="text">
        <string>Clean libs on device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editRulesFilePushButton">
       <property name="text">
        <string>Edit Rules File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="installQASI">
       <property name="toolTip">
        <string>Choose and install Ministro system wide Qt shared libraries.
This option is useful when you want to try your application on devices which don't have Android Market (e.g. Android Emulator).</string>
       </property>
       <property name="text">
        <string>Install Ministro, system-wide Qt shared libraries installer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chooseButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Choose APK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>installQASI</sender>
   <signal>toggled(bool)</signal>
   <receiver>chooseButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deployQtLibs</sender>
   <signal>clicked(bool)</signal>
   <receiver>useLocalQtLibs</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
